features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/application/application-keys/template.jag

```diff
@@ -169,15 +169,24 @@
 		  <div class="col-md-12 col-lg-9">
 		  	  {{> grants this}}
 		  	  {{> callback_url this}}
-			  <div class="form-group">	    
-			    <label><%=i18n.localize("Access token validity period")%> </label>
-			    <div class="input-group col-sm-6">
-                  <input type="text" size="10" title="validity_time" value='{{ValidityTime}}' class="form-control form-control-md validity_time">
-                  <div class="input-group-addon"><%=i18n.localize("Seconds.")%></div>
+			  <form class="add-margin-top-5x well">

```

Comment:
Shall we do the correct formatting for <form> tag in a way that it is aligned with the closing </form> tag?